src/query/storage/m3/cluster.go

```diff
@@ -149,26 +156,6 @@ type ClusterNamespaceDownsampleOptions struct {
 // ClusterNamespaces is a slice of ClusterNamespace instances.
 type ClusterNamespaces []ClusterNamespace
 
-// ClusterNamespacesByResolutionAsc is a slice of ClusterNamespace instances is
-// sortable by resolution.
-type ClusterNamespacesByResolutionAsc []ClusterNamespace
-
-func (a ClusterNamespacesByResolutionAsc) Len() int      { return len(a) }
-func (a ClusterNamespacesByResolutionAsc) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
-func (a ClusterNamespacesByResolutionAsc) Less(i, j int) bool {
-	return a[i].Options().Attributes().Resolution < a[j].Options().Attributes().Resolution
-}
-
-// ClusterNamespacesByRetentionAsc is a slice of ClusterNamespace instances is
-// sortable by retention.
-type ClusterNamespacesByRetentionAsc []ClusterNamespace
-
-func (a ClusterNamespacesByRetentionAsc) Len() int      { return len(a) }
-func (a ClusterNamespacesByRetentionAsc) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
-func (a ClusterNamespacesByRetentionAsc) Less(i, j int) bool {
-	return a[i].Options().Attributes().Retention < a[j].Options().Attributes().Retention
-}
```

Comment:
Moved these to src/query/storage/m3/cluster_resolver.go where they can be package-private (and renamed to `resolvedNamespacesBy*`).